server/controllers/index.js

```diff
@@ -1,9 +1,11 @@
 const profiles = require('./profile.controller');
 const users = require('./user.controller');
 const autorization = require('./auth.controller');
+const activation = require('./activation.controller');
 
```

Comment:
ok, i will remove its from index...thank)) I mounted it as usual)), and didn't consider its need...